scripts/set-up-workload-identity-configs.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+if [ "${GCP_PROJECT:-"unset"}" == "unset" ]; then
+  echo 'Error: GCP_PROJECT is unset. Please provide the GCP_PROJECT env variable to specify your GCP project' >&2
+  exit 1
+fi
+
+if [ "${FLEET_HOST_PROJECT:-"unset"}" == "unset" ]; then
+  echo 'Error: FLEET_HOST_PROJECT is unset. Please provide the FLEET_HOST_PROJECT env variable to specify your fleet host project' >&2
+  exit 1
+fi
+
+echo "Enabling the gkehub service API for project '${GCP_PROJECT}'"
+gcloud services enable gkehub.googleapis.com --project="${GCP_PROJECT}"
```

Comment:
This is enabled in the [terraform config](https://github.com/GoogleContainerTools/kpt-config-sync/pull/136/files#diff-26a3e2b80cf6d965b7f566230ae82bb5be9f5b209430a00a2396e0ccea67e6c3R19)